File: iosgeneratedcode/jcrystal/mobile/entities/enums/Size.swift
public enum Size : Int{
	case XS = 1000
	case S = 2000
	case M = 3000
	case L = 4000
	case XL = 5000
	var rawName: String?{
		switch self{
			case .XS : return "XS"
			case .S : return "S"
			case .M : return "M"
			case .L : return "L"
			case .XL : return "XL"
		}
	}
	var id: Int{
		return self.rawValue
	}
	public static func fromId(_ id : Int) -> Size{
		return Size(rawValue: id)!
	}
	static let values = [XS, S, M, L, XL]
}

File: src/main/java/dev/jcrystal/crystalfash/entities/Size.java
package dev.jcrystal.crystalfash.entities;

public enum Size {
	XS(1000),
	S(2000),
	M(3000),
	L(4000),
	XL(5000);
	public final long id;
	private Size(long id) {
		this.id = id;
	}
	public static Size fromId(long id) {
		for (Size val: Size.values()) {
			if (val.id==id) {
				return val;
			}
		}
		return null;
	}
}
